cirq/ops/common_channels.py

```diff
@@ -265,55 +288,75 @@ def _value_equality_values_(self):
         return self._p
 
     def __repr__(self) -> str:
-        return 'cirq.depolarize(p={!r})'.format(self._p)
+        if self._n_qubits == 1:
+            return f"cirq.depolarize(p={self._p})"
+        return f"cirq.depolarize(p={self._p},n_qubits={self._n_qubits})"
 
     def __str__(self) -> str:
-        return 'depolarize(p={!r})'.format(self._p)
+        if self._n_qubits == 1:
+            return f"depolarize(p={self._p})"
+        return f"depolarize(p={self._p},n_qubits={self._n_qubits})"
 
     def _circuit_diagram_info_(self,
                                args: 'protocols.CircuitDiagramInfoArgs') -> str:
+        if self._n_qubits == 1:
+            if args.precision is not None:
+                return f"D({self._p:.{args.precision}g})"
+            return f"D({self._p})"
         if args.precision is not None:
-            f = '{:.' + str(args.precision) + 'g}'
-            return 'D({})'.format(f).format(self._p)
-        return 'D({!r})'.format(self._p)
+            return f"D({self._p:.{args.precision}g},{self._n_qubits})"
+        return f"D({self._p},{self._n_qubits})"
 
     @property
     def p(self) -> float:
         """The probability that one of the Pauli gates is applied.
 
-        Each of the Pauli gates is applied independently with probability p / 3.
+        Each of the Pauli gates is applied independently with probability
+        p / (2**n_qubits - 1).
         """
         return self._p
 
+    @property
+    def n_qubits(self) -> int:
+        """The number of qubits
+        """
```

Comment:
All quotes should be on the same line, I think (though the formatter takes precedence over my opinion!).